uui-components/src/pickers/PickerBodyBase.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
 import isEqual from 'lodash.isequal';
 import { DataSourceListProps, DataSourceState, IEditable, IHasRawProps, isMobile } from '@epam/uui-core';
+import { ControlSize } from "../types";
 
 export interface PickerBodyBaseProps extends DataSourceListProps, IEditable<DataSourceState>, IHasRawProps<HTMLDivElement> {
     onKeyDown?(e: React.KeyboardEvent<HTMLElement>): void;
-    renderNotFound?: () => React.ReactNode;
+    renderNotFound?: (searchSize?: ControlSize) => React.ReactNode;
```

Comment:
i think we don't need searchSize parameter, you can use just component props.size prop